terraform/development/main.tf

```diff
@@ -49,19 +76,60 @@ module "networking" {
   vpc_flow_cloudwatch_log_expiration_days = 90
 }
 
-module "redis" {
-  source = "../modules/elasticache"
+module "parameter_store" {

```

Comment:
I'm not a fan of having to list all the params out like this (just makes this file really long), but not thought of a better way yet (prioritised getting ECS working)

Reply:
Since refactoring this is no longer valid - resolving this 😀